src/Raven.Server/Rachis/RachisConsensus.cs

```diff
@@ -1411,8 +1411,7 @@ public unsafe bool RemoveEntryFromRaftLog(long index)
                     var noopCmd = new DynamicJsonValue
                     {
                         ["Type"] = $"Noop for {Tag} in term {term}", 
-                        ["Command"] = "noop", 
-                        [nameof(CommandBase.UniqueRequestId)] = Guid.NewGuid().ToString()
```

Comment:
This is no longer unique operation?

Reply:
we don't really remove the command, we replace it with this `noop`, so the `noop` gets the guide of the deleted command

Reply:
Oh, I see